public/src/utils/models.tsx

```diff
@@ -287,6 +287,7 @@ export const countries: CommonStringObject = {
   ZW: 'Zimbabwe',
 };
 
+export const countryNames = Object.entries(countries).map(_ => [_[0], _[1]]);
```

Comment:
This is quite hard to follow - how about removing this and instead in `MultiSelectCountryEditor.tsx` define:
`const options: Option[] = Object.entries(countries).map(([value, label]) => ({ value, label }));`